src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3 } from './text';
+import Input from './input';
+import Button from './button';
+import Select from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessState from './success-state';
+
+const INPUT_BOX_WIDTH = '335px';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledSelect = styled(Select)`
+    color: ${colorMap.greyLightTwo};
+`;
+
+const ErrorMessage = styled(P)`
+    color: ${colorMap.salmon};
+`;
+
+const StyledSectionText = styled(ArticleH3)`
+    color: ${colorMap.greyLightTwo};
+    font-family: akzidenz;
+    font-weight: normal;
+`;
+
+const InstructorSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+        grid-row-gap: ${size.mediumLarge};
+        justify-content: space-between;
+        margin-bottom: ${size.xlarge};
+    }
+`;
+
+const InstitutionSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
```

Comment:
```suggestion
        grid-template-columns: 100%;
```


Since we are using mobile, maybe just using 100% here makes sense since (after we apply margins, etc. later in the page layout) we can just flow the whole width of the screen. Setting to 335px actually will overflow the screen on some smaller devices. Here is an iphone 5 for example:

![Screen Shot 2020-05-26 at 10 35 14 AM](https://user-images.githubusercontent.com/9064401/82913798-be6ead80-9f3c-11ea-8a5b-38f377b49545.png)